extensions-contrib/opencensus-extensions/src/main/java/org/apache/druid/data/input/opencensus/protobuf/OpenCensusProtobufInputRowParser.java

```diff
@@ -48,19 +51,31 @@
   private static final Logger LOG = new Logger(OpenCensusProtobufInputRowParser.class);
 
   private static final String SEPARATOR = "-";
-  public static final String NAME = "name";
-  public static final String VALUE = "value";
-  public static final String TIMESTAMP_COLUMN = "timestamp";
+  private static final String DEFAULT_METRIC_DIMENSION = "name";
+  private static final String VALUE = "value";
+  private static final String TIMESTAMP_COLUMN = "timestamp";
+  private static final String DEFAULT_RESOURCE_PREFIX = "resource.";
   private final ParseSpec parseSpec;
   private final List<String> dimensions;
 
+  private final String metricDimension;
+  private final String metricLabelPrefix;
+  private final String resourceLabelPrefix;
+
   @JsonCreator
   public OpenCensusProtobufInputRowParser(
-      @JsonProperty("parseSpec") ParseSpec parseSpec
+      @JsonProperty("parseSpec") ParseSpec parseSpec,
+      @JsonProperty("metricDimension") String metricDimension,
+      @JsonProperty("metricLabelPrefix") String metricPrefix,
+      @JsonProperty("resourceLabelPrefix") String resourcePrefix
   )
   {
     this.parseSpec = parseSpec;
     this.dimensions = parseSpec.getDimensionsSpec().getDimensionNames();
+    this.metricDimension = !Strings.isNullOrEmpty(metricDimension) ? metricDimension : DEFAULT_METRIC_DIMENSION;
```

Comment:
This ternary expression seems a bit convoluted. Can we remove the negation and flip the expressions to make it easier to read?

Reply:
Done.